site/en/docs/privacy-sandbox/attribution-reporting/understanding-noise/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Understanding noise'
+subhead: >
+  What noise means, where it is added, and how it impacts your measurement efforts.
+description: >
+  What noise means, where it is added, and how it impacts your measurement efforts.
+date: 2022-06-23
+updated: 2022-06-23
+authors:
+  - maudn
+---
+
+Data in aggregatable reports is _noised_; in other words, for a small percentage of cases, random data is sent instead of real reports/data. This practice preserves user privacy by preventing the joining of user identity across sites. 
+
+
+## Understanding noise in general
+
+While adding noise isn’t usually a part of ads measurement today, in many cases the noise added won’t substantially change how you interpret your results. 
+
+It may help to think about it in the following way:
+Would you be confident taking a certain decision based on certain data if that data wasn't noisy? 
```

Comment:
```suggestion
It may help to think about it in the following way:
Would you be confident taking a decision based on a certain piece of data if that data wasn't noisy? 
```